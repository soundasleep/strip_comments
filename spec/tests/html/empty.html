<!DOCTYPE html>
<html>
<head>
  <!--
  <style>* { margin: 0; padding: 0; }</style>
  -->
  <style>
    /* note comments within CSS is not stripped out */
  </style>
</head>
<body><!-- intentionally empty --></body>
</html>