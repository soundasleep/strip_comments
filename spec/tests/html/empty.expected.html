<!DOCTYPE html>
<html>
<head>
  
  <style>
    /* note comments within CSS is not stripped out */
  </style>
</head>
<body></body>
</html>